tests/e2e/kubetest2-kops/deployer/build.go

```diff
@@ -84,14 +123,7 @@ func (d *deployer) verifyBuildFlags() error {
 		d.KopsBaseURL = strings.Replace(d.StageLocation, "gs://", "https://storage.googleapis.com/", 1)
 	}
 
-	fi, err := os.Stat(d.KopsRoot)
-	if err != nil {
-		return err
-	}
-	if !fi.Mode().IsDir() {
-		return errors.New("--kops-root must be a directory")
-	}
-	if d.KopsVersionMarker != "" {
+	if d.KopsVersionMarker != "" && !d.BuildOptions.BuildKubernetes {
```

Comment:
Should it be like this?
```suggestion
	if d.KopsVersionMarker != "" && d.BuildOptions.BuildKubernetes {
```

Reply:
No.

This valid combination would fail `--build-kubernetes=true   --kops-version-marker=https://storage.googleapis.com/kops-ci/bin/latest-ci.txt`

Reply:
Then maybe the message is not correct?